### DISCUSSION TITLE
Signaling with custom parameters

### DISCUSSION THREAD

**Discussion by andreapier** [elsa-workflows/elsa-core#2468] [Q&A]
Hi, is it possible to read arbitrary parameters attached to the signal url generated by signalUrl JS function?
Something like: `http://localhost:28087/signals/trigger/<ELSA_GENERATED_TOKEN>?myParameter=myValue`

**Comment by sfmskywalker** [comment]
That should be possible, since that's an HTTP request. As long as the workflow executes within the HTTP context, you can access its querystring parameters using e.g. JavaScript: https://elsa-workflows.github.io/elsa-core/docs/next/expressions/expressions-javascript#querystring
